apps/librepcb/main.cpp

```diff
@@ -205,7 +205,22 @@ static bool isFileFormatStableOrAcceptUnstable() noexcept {
  ******************************************************************************/
 
 static FilePath determineWorkspacePath() noexcept {
-  FilePath wsPath(Workspace::getMostRecentlyUsedWorkspacePath());
+  static const QString FORCE_VAR_NAME("LIBREPCB_FORCE_WORKSPACE");
```

Comment:
I think I'd prefer the variable to be `LIBREPCB_WORKSPACE` (there's no "forcing" involved, it's just another configuration option that takes precedence), but I don't know what @ubruhin thinks about it.

Reply:
Agreed, LIBREPCB_WORKSPACE sounds better :) Wanna resolve this or wait for @ubruhin ?

Reply:
:+1: for LIBREPCB_WORKSPACE :)